src/jarabe/model/shell.py

```diff
@@ -668,8 +668,7 @@ def _active_window_changed_cb(self, screen, previous_window=None):
                 window = window.get_transient()
 
         act = self._get_activity_by_xid(window.get_xid())
-        if act is not None:
-            self._set_active_activity(act)
+        self._set_active_activity(act)
```

Comment:
This commit breaks the journal button below the user in the home view.  When this commit is applied, the button is ineffective.  When the commit is reverted, the button causes the journal to become visible (as expected).

Tested on:

- startx with a `.xinitrc` of `echo sugar | /home/saam/sugar-build/osbuild/run`
- running the sugar command within existing GNOME desktop